extensions/arc/package.json

```diff
@@ -601,37 +609,69 @@
                           "type": "kube_storage_class",
                           "required": true
                         },
-                        {
-                          "label": "%arc.cores-request.label%",
-                          "description": "%arc.sql.cores-request.description%",
-                          "variableName": "AZDATA_NB_VAR_SQL_CORES_REQUEST",
-                          "type": "number",
-                          "min": 1,
-                          "required": false
-                        },
                         {
                           "label": "%arc.cores-limit.label%",
```

Comment:
Moved limit fields before request fields because in my test workflows, i found it more natural for a user to specify the limit before the request value

Reply:
Did it here because I was modifying this stuff anyway.

Reply:
imo request before limit makes more sense.  I'd expect the minimum to come before the maximum.

Reply:
Sure. Will make the request fields before limit fields.

Reply:
Done.